manifest.webapp

```diff
@@ -242,9 +241,8 @@
       "type": "io.cozy.jobs"
     },
     "accounts": {
-      "description": "Used to create a temporary account for transfers",
-      "type": "io.cozy.accounts",
-      "verbs": ["POST", "DELETE", "GET"]
+      "description": "Used to create a temporary account for transfers and to configure accounts through Harvest",
+      "type": "io.cozy.accounts"
```

Comment:
Also, we shoud use "verbs": ["ALL"]

Reply:
Why should we use verbs : all ? When not used, it is implicit, IMHO we should not add stuff that does not serve.

Reply:
> No need to translate it in french?

No as banks is pre-installed and we do not have any text ready for this. cc @fffflo : it could be handy someday to have permissions texts in French if we plan to have a platform where banks would be installed by the user.

Reply:
> Why should we use verbs : all ? When not used, it is implicit, IMHO we should not add stuff that does not serve.

I prefer to be explicit. Specially on permissions.